java/src/jmri/configurexml/TransitManagerXml.java

```diff
@@ -150,7 +150,6 @@ public void loadTransits(Element sharedTransits, Element perNodeTransits) {
         List<Element> transitList = sharedTransits.getChildren("transit");
         log.debug("Found {} transits", transitList.size());
         TransitManager tm = InstanceManager.getDefault(jmri.TransitManager.class);
-        tm.setDataListenerMute(true);
```

Comment:
IIRC, this listener mute/unmute and the similar one in SectionManagerXml loading was inserted about 2 years ago to resolve O(N^2) loading performance.  I think @petecressman was (starting to?) work with some configurations that were big enough that it was starting to matter.  I don't have any more detail than that; the PR doesn't mention this, and my notes on the topic are pretty sparse. @petecressman do you remember this?  Do you have a file(s) that can test if this has a performance impact?  Thanks.

Reply:
Prior to this PR, the only ManagerDataListener I could find is the AbstractProxyManager. Everything else, including all GUI listeners, listened to the "length" or "beans" properties of the Managers, which were never muted, so I don't think muting this had the performance effects you are anticipating.